src/platforms/dart/usage/advanced-usage.mdx

```diff
@@ -52,6 +52,48 @@ try {
 }
 ```
 
+## Reporting bad HTTP requests as errors
+
+The `SentryHttpClient` can also catch exceptions that may occur during requests
+such as [`SocketException`](https://api.dart.dev/stable/2.13.4/dart-io/SocketException-class.html)s.
```

Comment:
```suggestion
The `SentryHttpClient` can also catch exceptions that may occur during requests,
such as [`SocketExceptions`](https://api.dart.dev/stable/2.13.4/dart-io/SocketException-class.html).
```

Reply:
we can't rename that, it's a class name

Reply:
ah okay, i assumed it was a typo because it looked very odd. how about this instead:

```suggestion
The `SentryHttpClient` can also catch exceptions that may occur during requests — for example [`SocketException`](https://api.dart.dev/stable/2.13.4/dart-io/SocketException-class.html).
```

Reply:
looks good, thanks